AuRoRA/src/vcs/vcs/pump.py

```
@@ -258,59 +403,94 @@ def retrieve_conduit_map_blueprint_from_hardware_spec() -> Dict[str, Any]:
-            temperature_gpu_str = f"{temperature_gpu}°C" if temperature_gpu is not None else "N/A"
-            print(f"Run #: {glob['run']+1} | Timestamp: [{glob['timestamp']}]")
-            print(f" CPU Load: {cpu_load_str} | CPU: {cpu_color}{temperature_cpu_str}\033[0m")
-            print(f" GPU Load: {gpu_load_str} | GPU: {gpu_color}{temperature_gpu_str}\033[0m")
-                        
-            time.sleep(0.1)
+        # Start the diagnosis
+        logger.info("Diagnosing VTC Pump Module...")
+        glob = {"timestamp": 0, "duration": 0.0, "run": 0, "glob": {}}
+    
+        # Initialize the vitals containers
+        cpu_loads = {}; cpu_temps = {}
```

Ruff (0.14.14)

[error] Multiple statements on one line (semicolon)

(E702)